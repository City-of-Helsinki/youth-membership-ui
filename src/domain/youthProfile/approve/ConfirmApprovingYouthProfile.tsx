import React from 'react';
import { useTranslation } from 'react-i18next';

import Text from '../../../common/components/text/Text';
import styles from './confirmApprovingYouthProfile.module.css';

function ConfirmApprovingYouthProfile() {
  const { t } = useTranslation();

  return (
    <div className={styles.container}>
      <Text variant="h1">{t('approval.approvedTitle')}</Text>
      <Text variant="info">{t('approval.approvedMessage')}</Text>
    </div>
  );
}
export default ConfirmApprovingYouthProfile;
