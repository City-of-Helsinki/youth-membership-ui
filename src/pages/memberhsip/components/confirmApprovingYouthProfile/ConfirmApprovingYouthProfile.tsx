import React from 'react';
import { useTranslation } from 'react-i18next';

import styles from './ConfirmApprovingYouthProfile.module.css';

function ConfirmApprovingYouthProfile() {
  const { t } = useTranslation();

  return (
    <div className={styles.container}>
      <h1>{t('approval.approvedTitle')}</h1>
      <p>{t('approval.approvedMessage')}</p>
    </div>
  );
}
export default ConfirmApprovingYouthProfile;
