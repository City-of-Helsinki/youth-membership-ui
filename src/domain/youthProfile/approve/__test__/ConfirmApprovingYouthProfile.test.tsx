import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ConfirmApprovingYouthProfile from '../ConfirmApprovingYouthProfile';

test('matches snapshot', () => {
  const wrapper = shallow(<ConfirmApprovingYouthProfile />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
