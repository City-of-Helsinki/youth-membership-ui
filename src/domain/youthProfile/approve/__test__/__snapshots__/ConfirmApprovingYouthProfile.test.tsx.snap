// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="container"
>
  <h1>
    J채ss채rihakemus hyv채ksytty!
  </h1>
  <p>
    Kiitos kun varmistit nuorisopalveluiden j채senyyden.
  </p>
</div>
`;
